cmd/new-ui/v1beta1/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	common_v1beta1 "github.com/kubeflow/katib/pkg/common/v1beta1"
+	ui "github.com/kubeflow/katib/pkg/new-ui/v1beta1"
+)
+
+var (
+	port, host, buildDir, dbManagerAddr *string
+)
+
+func init() {
+	port = flag.String("port", "80", "The port to listen to for incoming HTTP connections")
+	host = flag.String("host", "0.0.0.0", "The host to listen to for incoming HTTP connections")
+	buildDir = flag.String("build-dir", "/app/build", "The dir of frontend")
+	dbManagerAddr = flag.String("db-manager-address", common_v1beta1.GetDBManagerAddr(), "The address of Katib DB manager")
+}
+
+func main() {
+	flag.Parse()
+	kuh := ui.NewKatibUIHandler(*dbManagerAddr)
+
+	log.Printf("Serving the frontend dir %s", *buildDir)
+	frontend := http.FileServer(http.Dir(*buildDir))
+	http.HandleFunc("/katib/", kuh.ServeIndex(*buildDir))
+	http.Handle("/katib/static/", http.StripPrefix("/katib/", frontend))
+
+	http.HandleFunc("/katib/fetch_experiments/", kuh.FetchAllExperiments)
+
+	http.HandleFunc("/katib/submit_yaml/", kuh.SubmitYamlJob)
+	http.HandleFunc("/katib/submit_hp_job/", kuh.SubmitParamsJob)
+	http.HandleFunc("/katib/submit_nas_job/", kuh.SubmitParamsJob)
```

Comment:
I think we can leave one API with `/katib/create_experiment/` to create Experiments.

Reply:
Taking into consideration the other discussions around having a `CreateExperiment` function:
1. I'll remove the `/katib/submit_yaml/` route here
2. I'll remove the `/katib/submit_hp_job/` route here
3. I'll remove the `/katib/submit_nas_job/` route here
4. I'll add a `/katib/create_experiment/` route with the `kuh.CreateExperiment` function I'll create
5. [ui] Use the `create_experiment` route instead

Does this sound OK to you?

Reply:
Resolving this as we moved forward with this implementation